README.md

```diff
@@ -8,7 +8,13 @@ Enables greater flexibility in creating journeys through the website
 
 ### Dependencies
 
-* Mongo db (can use [dp-compose](https://github.com/ONSdigital/dp-compose) to stand up an instance in docker container, this requires docker)
+The follow dependencies can all be met by running the Auth stack and then logging into your local Florence using your sandbox Florence credentials:
```

Comment:
Is there an 'ing' missing at the end of the word 'follow' ?

Reply:
👍